Dockerfile

```diff
@@ -14,15 +14,15 @@ RUN /copy_runtime.sh
 RUN chmod +x /Kavita/Kavita
 
 #Production image
-FROM ubuntu:focal
+FROM ubuntu:oracular
```

Comment:
@DieselTech

Reply:
move to oracular because the include JPEG-XL in there, focal, requires custom build the support. Docker worked really fine. But didn't test ARM.

Reply:
Yes, adding MagicK was the last thing I wanted to do, problem is, while vips supports all the image formats, netvips binding and libraries are boundled with the standard version, not the ALL version.

Will test on ARM, tested on X64 docker, and windows without an issue. git said, every flavor is supported, but, yes,  testing will be needed for ARM.

I'm not going to enter the browser war, or format war, just adding the capability of supporting multiple formats and be dynamic about it, independent of the browser, seems the way to go.  And since, from my own testing JPEG-XL seems to be a techincal marvel.